SchoolHousing/SchoolHousing/Pages/Index.cshtml.cs

```diff
@@ -8,19 +8,66 @@
 using QuickType;
 using AffordableHouseResponse;
 using ElementarySchoolResponse;
+using Newtonsoft.Json.Schema;
+using Newtonsoft.Json.Linq;
+
 public class IndexModel : PageModel
 {
+    public string DownloadData(string jsonendpoint)
+    {
+        string rawData = "";
+        using (WebClient webClient = new WebClient())
+        {
+            rawData = webClient.DownloadString(jsonendpoint);
+        }
+        return rawData;
+    }
+
     public void OnGet()
     {
         using (var webClient = new WebClient())
         {
-            string schoolString = webClient.DownloadString("https://data.cityofchicago.org/resource/tj8h-mnuv.json");
+            IDictionary<long, AffordableHouseResponse.AffordableHouse> allHouses = new Dictionary<long, AffordableHouseResponse.AffordableHouse>();
+            string houseString = DownloadData("https://data.cityofchicago.org/resource/s6ha-ppgi.json");
+            AffordableHouseResponse.AffordableHouse[] afHouses = AffordableHouseResponse.AffordableHouse.FromJson(houseString);
+            foreach (AffordableHouseResponse.AffordableHouse house in afHouses)
+            {
+                if (allHouses.ContainsKey(house.ZipCode))
+                {
+                    //do nothing
```

Comment:
Skip this if not needed.  Basically, turn the test around (use a ! to turn a true to false, and vice-versa), and then only have an if part, not an else part.  An empty block is clutter.

Reply:
I completely agree with you. However, I just wanted to keep an empty block for the team only in case if they're going to set up an exception as per their requirement, this block could be useful.